README.md

```diff
@@ -1095,6 +1128,192 @@ VTP file name. Valid values are a string or the keyword 'default'.
 ##### `password`
 Password for the VTP domain. Valid values are a string or the keyword 'default'.
 
+--
+### NetDev Resource Type Details
+
+The following resources are listed alphabetically.
+
+--
+
+### Type: domain_name
+
+Configure the domain name of the device
+
+#### Parameters
+
+##### `ensure`
+Determines whether or not the config should be present on the device. Valid values are 'present' and 'absent'. Default value is 'present'.
+
+##### `name`
+Domain name of the device. Valid value is a string.
+
+### Type: name_server
+
+#### Parameters
+
+##### `ensure`
+Determines whether or not the config should be present on the device. Valid values are 'present' and 'absent'. Default value is 'present'.
+
+##### `name`
+Hostname or address of the DNS server.  Valid value is a string.
+
+### Type: network_dns
+
+#### Parameters
+
+##### `ensure`
+Determines whether or not the config should be present on the device. Valid
+values are 'present' and 'absent'. Default value is 'present'.
+
+##### `name`
+Name, generally "settings", not used to manage the resource.  Valid value is a string.
```

Comment:
I'm not sure what this is trying to say exactly...  Name not used to manage the resource?